src/applications/health-care-questionnaire/questionnaire/api/submit-questionnaire.js

```diff
@@ -14,15 +14,16 @@ const submit = (form, formConfig) => {
   //   eventData,
   // });
 
-  // return submitToUrl(
-  //   JSON.stringify(body),
-  //   formConfig.submitUrl,
-  //   formConfig.trackingPrefix,
-  //   eventData,
-  // );
-  return new Promise((resolve, _reject) => {
-    resolve(body);
-  });
+  return submitToUrl(
+    JSON.stringify(body),
+    formConfig.submitUrl,
+    formConfig.trackingPrefix,
+    // eventData,
+  );
+  // return new Promise((_resolve, reject) => {
+  //   // resolve(body);
+  //   reject(body);
+  // });
```

Comment:
Do we need to keep this commented out data?

Reply:
Yes, so that when we turn the error back off, it's right there.